src/store/store.ts

```diff
@@ -1,18 +0,0 @@
-import type { BrandingAppearance } from "src/networking/responses/branding-response";
-import { writable } from "svelte/store";
-
-export const DEFAULT_STORE_VALUES: BrandingAppearance = {
-  color_buttons_primary: "",
-  color_accent: "",
-  color_error: "",
-  color_product_info_bg: "",
-  color_form_bg: "",
-  color_page_bg: "",
-  font: "",
-  shapes: "default",
-  show_product_description: false,
-};
-
-// Store to hold appearance configuration object for custom overrides
-export const appearanceConfigStore =
-  writable<BrandingAppearance>(DEFAULT_STORE_VALUES);
```

Comment:
I think the intention behind using a store was to avoid passing the appearence around the widgets, but it's true that's basically changing from explicit dependencies to implicit, so I think this makes sense 👍

Reply:
In the end we need to pass it only to:
1. The parent HTML element (Aside or Main) and the variables will be applied downwards
2. The purchase form since we need it to customize the stripe elements

There's not a lot of drill-down